bundle/DataCollector/GitDataCollector.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\SiteBundle\DataCollector;
+
+use SebastianFeldmann\Git\Repository;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\DataCollector\DataCollector;
+
+final class GitDataCollector extends DataCollector
+{
+    public function collect(Request $request, Response $response, \Exception $exception = null)
+    {
+        $repository = new Repository();
+
+        $branch = $repository->getInfoOperator()->getCurrentBranch();
```

Comment:
Can we add info if the current branch is tracking a remote branch?

For example:

`1.11 (origin/1.11)`

Reply:
Not in an easy way (using provided library).
It would require some custom parsing..

Reply:
Then merging as is. Good enough :)